<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game of life</string>
  </property>
  <widget class="QWidget" name="mainwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QPushButton" name="resetTableButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>139</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset table</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="size">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>197</y>
      <width>139</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Size</string>
    </property>
    <widget class="QPlainTextEdit" name="tableRowCountTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>25</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="tableColumnCountTextBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="plainText">
      <string>25</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Row count</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Column count</string>
     </property>
    </widget>
    <widget class="QPushButton" name="changeTableSizeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change size</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="main">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>139</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Main</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="timerIntervalSlider">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="sliderPosition">
        <number>250</number>
       </property>
       <property name="tracking">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="invertedControls">
        <bool>false</bool>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBothSides</enum>
       </property>
       <property name="tickInterval">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startGenerationButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stepGenerationButton">
       <property name="text">
        <string>Step</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopGenerationButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="currentTime">
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>159</x>
      <y>9</y>
      <width>861</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="workSpaceLayout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
